dotCMS/src/main/java/com/dotcms/contenttype/business/FieldAPIImpl.java

```diff
@@ -169,6 +180,11 @@ public void delete(final Field field, final User user) throws DotDataException,
 
           fieldFactory.moveSortOrderBackward(oldField.sortOrder());
           fieldFactory.delete(field);
+
+          ActivityLogger.logInfo(ActivityLogger.class, "Delete Field Action",
+                  String.format("User %s/%s eleted field %s from %s Content Type.", user.getUserId(), user.getFirstName(),
+                          field.name(), structure.getName()));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "structure" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)